public/css/partials/_bootstrap-overrides.scss

```diff
@@ -1,10 +1,3 @@
-.btn {
```

Comment:
Yeah because I wanted to use the default buttons. There were too many primaries.